pkg/boot/kexec/memory_linux.go

```diff
@@ -667,3 +667,52 @@ func (m *MemoryMap) Insert(r TypedRange) {
 	newMap.sort()
 	*m = newMap
 }
+
+// Memory type (uint32) in UefiPayload
+const (
+	PayloadTypeRAM      = 1
+	PayloadTypeDefault  = 2
+	PayloadTypeAcpi     = 3
+	PayloadTypeNVS      = 4
+	PayloadTypeReserved = 5
+)
+
+// payloadMemoryMapEntry represent a memory map entry in payload param
+type payloadMemoryMapEntry struct {
+	Start uint64
+	End   uint64
+	Type  uint32
+}
+
+// PayloadMemoryMapParam is payload's MemoryMap parameter
+type PayloadMemoryMapParam []payloadMemoryMapEntry
+
+var rangeTypeToPayloadMemType = map[RangeType]uint32{
+	RangeRAM:      PayloadTypeRAM,
+	RangeDefault:  PayloadTypeDefault,
+	RangeACPI:     PayloadTypeAcpi,
```

Comment:
nit: keep Go readability standards. I.e. acronyms should be all upper case, PayloadTypeACPI https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Fixed. Thanks.